items/test_models.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+from .models import Item, Group
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.db import IntegrityError
+
+
+class ItemModelTest(TestCase):
```

Comment:
this file in you PR. PR - is what exactly you are going to merge to master. It's a diff between your branch and master. So it's your code or you failed with merging code from master to your branch